catacombs-chimeras-service/src/test/java/cz/muni/fi/service/TroopServiceImplTest.java

```diff
@@ -267,4 +268,85 @@ public void testTroopsAvgExperienceReportNoTroops() {
         assertThat(troopReport.getLowestAverageExperience(), is(0.0));
         assertThat(troopReport.getTroops(), hasSize(0));
     }
+    
+    @Test
+    public void testGetMoneyPerHeroList() {
+        Hero heroA = new Hero("A");
+        heroA.setId(1L);
+        Hero heroB = new Hero("B");
+        heroB.setId(2L);
+        Hero heroC = new Hero("C");
+        heroC.setId(3L);
+        Hero heroD = new Hero("D");
+        heroD.setId(4L);
+        Hero heroE = new Hero("E");
+        heroE.setId(5L);
+        Hero heroG = new Hero("G");
+        heroG.setId(6L);
+        
+        Troop alfa = new Troop("alfa");
+        alfa.setId(1L);
+        alfa.setAmountOfMoney(2L);
+        alfa.addHero(heroA);
+        alfa.addHero(heroB);
+        alfa.addHero(heroC);
+        
+        Troop beta = new Troop("beta");
+        beta.setId(2L);
+        beta.setAmountOfMoney(120L);
+        beta.addHero(heroD);
+        beta.addHero(heroE);
+
+        Troop gamma = new Troop("gamma");
+        gamma.setId(3L);
+        gamma.setAmountOfMoney(150L);
+        gamma.addHero(heroG);
+        
+        final List list = Arrays.asList(alfa, beta, gamma);
+        when(troopDao.findAll()).thenReturn(list);
+        
+        TroopWealthDTO troopWealth = troopService.getMoneyPerHeroList();
+        assertThat(troopWealth.getTroopWealthList().size(), is(equalTo(3)));
+        assertThat(troopWealth.getTroopWealthList().get(2).getMoneyPerHero(), is(new Long(10000)));
```

Comment:
jo já sem tam měl předtím jiný čísla a  tu sem to neopravil. dělal sem to ve spěchu